File: Python/Factorial.py
x=int(input("Enter Number: "))
p=1
i=1
if x<0:
    print("Enter valid Number -ve value is not valid")
if x==0:
    print("Factorial is 1")
else:
    while i <= x:
        p=p*i
        i=i+1
        print("Factorial is ",p)

File: Java/src/factorial.java
import java.util.*;
public class factorial {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the number for the factorial: ");
        int num = sc.nextInt();

        int pro = 1;
        for(int i=2;i<=num;i++){
            pro*=i;
        }
        System.out.printf("Factorial of %d is %d",num,pro);

    }
}
